app/lib/dags/host_filter.json.erb

```diff
@@ -5,6 +5,7 @@
     <% if @attribute_dict[:fastq2] %>
       "fastqs": ["<%= @attribute_dict[:fastq1] %>", "<%= @attribute_dict[:fastq2] %>"],
       "star_out": ["unmapped1.<%= @attribute_dict[:file_type] %>", "unmapped2.<%= @attribute_dict[:file_type] %>"],
+      "srst2_out": ["output.log", "output__genes__ARGannot_r2__results.txt", "output__fullgenes__ARGannot_r2__results.txt"],
```

Comment:
Since this line is the same in both the `if` and the `else` clause, you should take it out of the conditional to make the code more maintainable. Otherwise, any change to srst2 output files will require 2 lines of code to be modified instead of 1.

Reply:
for sure, TODO: remove from here